alchemist/alchemist-cognitive-agents/build.gradle.kts

```diff
@@ -11,6 +11,7 @@ dependencies {
     implementation(project(":alchemist-time"))
     implementation(project(":alchemist-implementationbase"))
     implementation(project(":alchemist-influence-sphere"))
+    compile("com.uchuhimo:konf:0.13.3")
```

Comment:
This must be fixed.
1. `compile` is deprecated. It's either `api` or `implementation`
2. Versions and libraries be dealt with by using buildSrcVersions